pkg/apis/core/v1/defaults.go

```diff
@@ -182,29 +182,6 @@ func SetDefaults_Pod(obj *v1.Pod) {
 				}
 			}
 		}
-		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) &&
-			obj.Spec.Containers[i].Resources.Requests != nil {
-			// For normal containers, set resize restart policy to default value (NotRequired), if not specified.
-			resizePolicySpecified := make(map[v1.ResourceName]bool)
-			for _, p := range obj.Spec.Containers[i].ResizePolicy {
-				resizePolicySpecified[p.ResourceName] = true
-			}
-			setDefaultResizePolicy := func(resourceName v1.ResourceName) {
```

Comment:
can drop the "defaulting" in the fuzzer as well now, right?

Reply:
Yeah, but that change was rolled back in https://github.com/kubernetes/kubernetes/pull/128875. We just won't add it back when we re-enable IPPVS.